### DISCUSSION TITLE
Исправить нейминг в EntityStateBuilder

### DISCUSSION THREAD

**Discussion by saturov** [surfstudio/SurfGear#180] [Ideas]
Предлагаю внести следующие изменения в `EntityStateBuilder`.

- Переименовать файл, в котором он описан. Сейчас файл называется `entity_stream_builder.dart`, а должен `entity_state_builder.dart`, по аналогии с аналогичными файлами. Наименование файла должно соответствовать имени класса, который в нем содержится.
- Поле класса `child` переименовать в `builder`. Текущее наименование нарушает семантику. Все поля класса типа `DataWidgetBuilder<T>` имеют постфикс `...Builder` в наименовании (например, `loadingBuilder`).

**Comment by r3tam** [comment]
Так и есть, надо сделать. По моему есть ишью уже на это

**Comment by grafovdenis** [comment]
* Сейчас файл называется правильно - [entity_state_builder.dart](https://github.com/surfstudio/SurfGear/blob/main/packages/relation/lib/src/builder/entity_state_builder.dart).
* Согласен, могу поправить в ближайшее время.
